kodi.py

```diff
@@ -91,11 +91,14 @@
 def sanitize_name(media_name):
   # Normalize string
   normalized = unicodedata.normalize('NFKD', media_name).encode('ASCII', 'ignore')
-  # Strip symbols
-  m = RE_NAME_WITH_PARAM.match(normalized)
-  if m:
-    return m.group(1)
-  return media_name
+
+  # Strip things between and including brackets and parenthesis
+  removed_paren = re.sub(r'\([^)]*\)', '', normalized)
+  removed_bracket = re.sub(r'\[[^)]*\]', '', removed_paren)
```

Comment:
Plus, scrapers usually add those suffixes to disambiguate one title from another.  What would a user do if they had both "The Office (US)" and "The Office (UK)"?  Or "Archer (2009)" and "Archer (1975)"?